### DISCUSSION TITLE
Why was Numpy minimum version changed to 1.23?

### DISCUSSION THREAD

**Discussion by JulienGrv** [danielhrisca/asammdf#726] [Q&A] [answered]
Hi @danielhrisca,

I just saw that in the release 7.1.0 was the support of Python version 3.7 dropped because of the minimum Numpy version being now 1.23. It seems a bit early to me because Python version 3.7 will reach its [end-of-life not before next year](https://endoflife.date/python). Could you eventually detail why you had to update to Numpy version 1.23?

Thanks in advance for your answer and as always thank you for your dedication and hard work, Julien

**Comment by danielhrisca** [comment, marked as answer]
I had a lot of trouble getting the github actions to complete successfully, switching to the latest numpy/scipy finnaly cleared the issues. Are you still using Python 3.7?

**Reply by JulienGrv** [reply]
At my company, I know that we have a machine in our continuous integration running asammdf under Python version 3.7 which is why I asked the question. It's not an issue at the moment but if for any reason it becomes one, I will be able to investigate this by myself since now I know the reason why you dropped it in the first place. Thank you
